Makefile

```
@@ -68,13 +68,16 @@ migrate: venv  ## Migrate the database
 	$(WITH_VENV) python app/manage.py migrate
 
 .PHONY: generate_models
-generate_models: venv  ## Generate new app models.py file
-	$(WITH_VENV) python app/manage.py inspectdb --database milmove > new_models.py
-	mv new_models.py app/milmove_app/models.py
+generate_models: generate_models_only
 	pre-commit run --all-files black || true
 	pre-commit run --all-files fix-encoding-pragma || true
 	@echo "Ignore errors from pre-commit, they are expected"
 
+.PHONY: generate_models_only
+generate_models_only: venv  ## Generate new app models.py file
```

Generally we name these:

```suggestion
generate_models_standalone: venv  ## Generate new app models.py file
```

Also, I'd get rid of `venv` here. Meaning:

```suggestion
generate_models_standalone:  ## Generate new app models.py file
```

In CircleCI we don't need a virtualenv since we're inside a container. So leave `venv` on the `generate_models` target.

---

Just to make sure I'm following, if the venve is removed is there another mechanism that installs the python requirements? Looks like the venv and pip install are in a single target. https://github.com/transcom/milmove_admin/blob/06e3144d276850a275a18b90ae4c3d3084f05f0e/Makefile#L16

---

Yeah, you'll want to install the python requirements a different way in CircleCI. Not with `virtualenv`.